src/screens/HomeScreen.js

```
@@ -9,49 +9,49 @@ import {
-} from "react-native";
+} from 'react-native';
 
-import Papa from "papaparse";
-import { useIsFocused } from "@react-navigation/native";
-const { VpnServiceModule, MainActivity } = NativeModules;
-import { Buffer } from "buffer"; // Make sure to install buffer with `npm install buffer`
-import CustomSnackbar from "../components/CustomSnackbar";
+import Papa from 'papaparse';
+import {useIsFocused} from '@react-navigation/native';
+const {VpnServiceModule, MainActivity} = NativeModules;
+import {Buffer} from 'buffer'; // Make sure to install buffer with `npm install buffer`
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)